public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -76,7 +76,7 @@ function ContextMenu (uiContextMenu) {
     }; //handles both key down and key up events
 
     function checkRadioButton (value) {
-        uiContextMenu.radioButtons.filter(function() {return this.value == value}).prop("checked", true).trigger("click");
+        uiContextMenu.radioButtons.filter(function() {return this.value == value}).prop("checked", true).trigger("change");
```

Comment:
What was the purpose of this change?

Reply:
We were binding the change event in the first place so clicking was unnecessary. However, I will revert this and use the lowLevelLogging parameter to avoid any unknown side effects.